samples/ActionConstraintSample.Web/project.json

```diff
@@ -12,9 +12,12 @@
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*"
   },
   "frameworks": {
-    "net451": { },
-    "dnxcore50": {
-      "imports": "portable-net451+win8",
+    "net451": {},
+    "netstandardapp1.5": {
+      "imports": [
+        "dnxcore50",
+        "portable-net451+win8"
+      ],
```

Comment:
Could you explain what this means? Is it somewhat analogous to having this project support `net451`, `dnx451` and `dnxcore50`?

Also `dnxcore50` --- really?